cli/ballerina-cli/src/main/resources/cli-help/ballerina-persist.help

```diff
@@ -33,21 +31,38 @@ DESCRIPTION
                Department department;
            |};
 
-       Then, generate the client API and SQL script based on the data model. Use the generated
-       API to query and manipulate the persistent data in the application and use the generated
-       SQL script to create the database tables.
+       Then, generate the client API based on the data model. You can use the `bal build` command to generate the client.
+       This process will automatically regenerate the client whenever you modify the data model during project builds.
```

Comment:
```suggestion
       This process regenerates the client when modifying the data model during project builds.
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/42542/commits/10e8107d9a0e35db389b4d748ba1b8effc819468